### PR TITLE
adds personal portfolio activity 2

### PR DESCRIPTION
**Pull request by spitsfire**

### CODE REVIEW

**Review by CheezItMan — COMMENTED** [review]
An observation:

Some people might find the lack of structure here frustrating.  I don't know that we should change this, but something to note.

Maybe we could add a suggested list of properties to experiment with.

**Review by jericahuang — APPROVED** [review]

**Comment by spitsfire** [comment]
> An observation:
> 
> Some people might find the lack of structure here frustrating. I don't know that we should change this, but something to note.
> 
> Maybe we could add a suggested list of properties to experiment with.

Yeah very fair! We could definitely make a change to this specific page with some common css properties, or add them to the slide deck for the round table

File: styles-and-presentation/activity-personal-portfolio-2.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+In this second activity, we will style our personal portfolio pages with CSS, keeping in mind the power of classes and ids.
+
+## Activity Instructions
+
+### Requirements
+
+This portfolio should have at least one CSS stylesheet (likely named  `styles.css`). Each page may use the same stylesheet, or you can create distinct stylesheets for each page.
+
+There are no restrictions or requirements for what CSS properties you use. Follow your curiosity and style your personal portfolio any way you wish!
```
Comment (spitsfire) [inline styles-and-presentation/activity-personal-portfolio-2.md:15]:
```suggestion
There are no restrictions or requirements for what CSS properties you use; however, if you need a place to start MDN has a list of the [most commonly used CSS properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Properties_Reference) with visual examples. Follow your curiosity and style your personal portfolio any way you wish!
```

**Comment by spitsfire** [comment]
@CheezItMan I added a link to MDN's most commonly used CSS properties. Clicking on the property brings up a visual example. Do you think that's sufficient or should we make our own curated list?

**Comment by CheezItMan** [comment]
I think for now it's ok.  We can adjust based on student feedback/performance.

**Pull request merged**